icevision/models/fastai/unet/prediction.py

```diff
@@ -39,10 +39,21 @@ def convert_raw_predictions(
     keep_images: bool = False,
 ) -> List[Prediction]:
     mask_preds = raw_preds.argmax(dim=1)
-    tensor_images, *_ = batch
```

Comment:
Same comment as the previous file.
UNet is outside of `mmseg`.
Did anything break because of the changes to `MaskArray`, hence you were obliged to edit those files?